src/main/scala/is/hail/distributedmatrix/BlockMatrix.scala

```diff
@@ -749,12 +792,12 @@ class WriteBlocksRDD(irm: IndexedRowMatrix, path: String, gp: GridPartitioner) e
   def compute(split: Partition, context: TaskContext): Iterator[Int] = {
     val blockRowIndex = split.index
```

Comment:
There are a dozen or so instances of blockRowIndex / blockColIndex remaining in WriteBlocksRDD. These should all be changed to blockRow and blockCol. The comment just above `case class WriteBlocksRDDPartition` should be adjusted as well.